drivers/spi/spi_nrfx_spim.c

```diff
@@ -28,7 +28,7 @@ LOG_MODULE_REGISTER(spi_nrfx_spim, CONFIG_SPI_LOG_LEVEL);
 #endif
 
 /* Maximum chunk length (depends on the EasyDMA bits, equal for all instances) */
-#define MAX_CHUNK_LEN BIT_MASK(SPIM0_EASYDMA_MAXCNT_SIZE)
+#define MAX_CHUNK_LEN SPIM_TXD_MAXCNT_MAXCNT_Msk
```

Comment:
I would love to see here build assert that checks SPIM_TXD_MAXCNT_MAXCNT_Msk to equal SPIM_RXD_MAXCNT_MAXCNT_Msk. It might look as an overkill. I would just feel safe then as for me it seems we are using it for both directions.

Reply:
done